api/handlers.go

```diff
@@ -330,6 +330,39 @@ func getVolume(c *context, w http.ResponseWriter, r *http.Request) {
 	httpError(w, fmt.Sprintf("No such volume: %s", name), http.StatusNotFound)
 }
 
+func GetUsedVolumes(containers []*cluster.Container) map[string]bool {
```

Comment:
This function doesn't need to be public (i.e. it can be named `getUsedVolumes` instead of `GetUsedVolumes`.

Reply:
right, the unit test is in the same package, no need for public.
I will fix that